tests keep failing due to browserslist error reporting to stderr

## Type of Change
- [ ] New Feature Request
- [ ] Documentation / Website
- [ ] Improvement / Suggestion
- [ ] Bug
- [x] Other (please clarify below)

## Summary
Browserslist has [started outputting an error to `stderr`](https://github.com/browserslist/browserslist/issues/361) all of a sudden
```shell
$ yarn build
yarn run v1.12.3
$ yarn clean && node . build
$ rimraf ./**/.greenwood/** && rimraf ./**/public/** && rimraf ./coverage
-------------------------------------------------------
Welcome to Greenwood ♻️
-------------------------------------------------------
Reading project config
Initializing project workspace contexts
Generating graph of workspace files...
Scaffolding out project files...
Generate pages from templates...
Writing imports for md...
Writing Lit routes...
setup index page and html
Scaffolding complete.
Building project for production.
Building SPA from compilation...
Browserslist: caniuse-lite is outdated. Please run next command `yarn upgrade`
Browserslist: caniuse-lite is outdated. Please run next command `yarn upgrade`
Browserslist: caniuse-lite is outdated. Please run next command `yarn upgrade`
webpack build complete
...................................
Static site generation complete!
...................................
✨  Done in 24.70s.
```

which is causing error handling specs to fail because it is only expected Greenwood's output logging.
```shell
7 passing (1m)
  20 failing

  1) Build Greenwood With:
       Empty Configuration and Default Workspace
         "before all" hook in "Empty Configuration and Default Workspace":
     Error: the string "Browserslist: caniuse-lite is outdated. Please run next command `yarn upgrade`\n" was thrown, throw an Error :)
      at process._tickCallback (internal/process/next_tick.js:68:7)

  2) Build Greenwood With:
       Custom Configuration with a bad value for theme file
         should throw an error that themeFile must be a filename:
     AssertionError: expected 'Browserslist: caniuse-lite is outdated. Please run next command `yarn upgrade`\n' to include 'Error: greenwood.config.js themeFile must be a valid filename. got {} instead.'
      at Context.it (packages/cli/test/cases/build.config.error-theme-file/build.config.error-theme-file.spec.js:35:24)
      at process._tickCallback (internal/process/next_tick.js:68:7)

  3) Build Greenwood With:
       Custom Meta Configuration and Nested Workspace
         "before all" hook in "Custom Meta Configuration and Nested Workspace":
     Error: the string "Browserslist: caniuse-lite is outdated. Please run next command `yarn upgrade`\n" was thrown, throw an Error :)
      at process._tickCallback (internal/process/next_tick.js:68:7)

(etc, etc)
```

## Details
At least until we can land [updating `browserslist`](https://github.com/ProjectEvergreen/greenwood/issues/276) as its own PR, and just to protect ourselves down the road from it happening again, it looks like there is a solution to suppress this, as least for our tests and our users.  (leaving it on for development seems to make sense though).

Just need to add this
```shell
BROWSERSLIST_IGNORE_OLD_DATA=true
```

--- a/package.json
+++ b/package.json
@@ -20,8 +20,8 @@
     "build": "yarn clean && node . build",
     "serve": "yarn build && cd ./public && ws",
     "develop": "yarn clean && node . develop",
-    "test": "yarn clean && nyc mocha ./packages/**/test/**/**/*.spec.js --timeout 30000",
-    "test:tdd": "yarn clean && mocha --watch ./packages/**/test/**/**/*.spec.js --timeout 15000"
+    "test": "export BROWSERSLIST_IGNORE_OLD_DATA=true && yarn clean && nyc mocha ./packages/**/test/**/**/*.spec.js --timeout 30000",
+    "test:tdd": "export BROWSERSLIST_IGNORE_OLD_DATA=true  && yarn clean && mocha --watch ./packages/**/test/**/**/*.spec.js --timeout 15000"
   },
   "devDependencies": {
     "chai": "^4.2.0",

